### ISSUE TITLE
Updating notifications

### ISSUE THREAD

**Issue by krzysztof113** [HelloZeroNet/ZeroNet#2335]
### Step 1: Please describe your environment

  * ZeroNet version: 3868,3870
  * Operating system: Linux
  * Web browser: na
  * Tor status: not available/always/disabled
  * Opened port: yes/no
  * Special configuration: na

### Step 2: Describe the problem:
There is a problem with updating from older version. The message on Hello page and Update page shows misleading information that Zeronet bundle is up to date. Both Hello and Update page should provide information that manual update actions are required. Also link to instructions. 
If user doesn't follow Zeronet development is not even aware of the newer version and potential security issues of older version.

#### Steps to reproduce:
Check Hello Zeronet page and http://127.0.0.1:43110/1UPDatEDxnvHDo7TXvq6AEBARfNkyfxsp/

![Zrzut ekranu z 2019-12-01 00-44-37](https://user-images.githubusercontent.com/11329331/69907567-644ebe80-13d7-11ea-82fc-1fb2d5197bd6.png)
![Zrzut ekranu z 2019-12-01 00-46-34](https://user-images.githubusercontent.com/11329331/69907568-69137280-13d7-11ea-94f5-1d82ccc12b07.png)

**Comment by filips123** [comment]
You have Python 2.7 version which you can't update automatically. You need to manually download ZeroBundle again, copy your `data` directory to it, and then use new version.

Yes, it would be good to provide details about this and maybe also support to update ZeroBundle completely with new Python version.

@HelloZeroNet It might be good to backport changes for updating complete ZeroBundle to Python 2.7 version so users will be able to download Python 3 from UI. Or at least add some notification/description for instructions.

**Comment by krzysztof113** [comment]
I have multiple versions. I believe that some percentage of the peers with 0.6.5 * version is because of lack knowledge about newer version.

**Comment by HelloZeroNet** [comment]
Yeah, I will add a notification about the new version. On http://127.0.0.1:43110/Stats I see 70% version 0.7.1, 20% 0.6.5 and 10% earlier version, so it's not that bad

**Comment by krzysztof113** [comment]
Related #2174
